test/mock_exec.ml

```diff
@@ -80,6 +82,24 @@ let closing redir fn =
        Lwt.return_unit
     )
 
+let exec_standalone_fetcher tmpdir _base =
+  let open Yojson.Basic in
+  (* Create a minimal manifest.json matching what the standalone fetcher
+     expects, as well as a config.json containing the environment. *)
+  let configfile = "config.json" in
+  let json = `List [ `String mock_environment ] in
+  let json = `Assoc [ ("Env", json) ] in
+  let json = `Assoc [ ("config", json ) ] in
+  json |> to_file (tmpdir / configfile);
+  let json = `Assoc [ ("Config", `String configfile) ;
+                      ("Layers", `List [ `String "base.tar" ]) ] in
+  let json = `List [ json ] in
+  json |> to_file (tmpdir / "manifest.json");
+  (* Copy the mock file archive. *)
+  let mydir = Sys.getcwd () in
+  let base_tar = mydir / "base.tar" in
+  Os.copy ~src:base_tar tmpdir >>= (fun () -> Lwt_result.return 0)
```

Comment:
Why is the ` >>= (fun () -> Lwt_result.return 0)` needed?

Reply:
Because `Os.copy` return an `Lwt` but `Lwt_result` is expected by the caller.